winui/TreeGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,595 @@
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+### Export column width to Excel
+
+You can export the columns with its actual width by setting the `CanExportColumnWidth` property to `true`. 
+
+{% tabs %}
+{% highlight c# %}
+var options =new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.CanExportColumnWidth = true;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+The default value of the `CanExportColumnWidth` property is `true`.
+
+### Export with customized row height
+
+You can export the TreeGrid to Excel with customized row height by using the `DefaultRowHeight` property.
+
+{% tabs %}
+{% highlight c# %}
+var options =new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.DefaultRowHeight = 60;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+### Change the node expand state in Excel
+
+You can change the node expanding state in Excel by using the `NodeExpandMode` property.
+
+{% tabs %}
+{% highlight c# %}
+var options =new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.NodeExpandMode = NodeExpandMode.CollapseAll;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+### Changing start row and column index while exporting
+
+Export the data to the specified row index and column index in the worksheet, by setting the `StartRowIndex` and `StartColumnIndex` properties.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.StartRowIndex = 3;
+options.StartColumnIndex = 3;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-change-row-column-index.png" alt="Changing Row and Column Index while Exporting from WinUI TreeGrid to Excel" width="100%" Height="Auto"/>
+
+## Row Height and Column Width customization  
+
+After exporting the data to Excel, you can set different row heights and column widths for the columns. You can refer to [here](http://help.syncfusion.com/file-formats/xlsio/worksheet-rows-and-columns-manipulation#adjust-row-height-and-column-width) for more information.

```

Comment:
After exporting the data to Excel, set different row heights and column widths for the columns. Refer to [here](http://help.syncfusion.com/file-formats/xlsio/worksheet-rows-and-columns-manipulation#adjust-row-height-and-column-width) for more information.

Reply:
updated